addons/website/data/image_library.xml

```diff
@@ -709,5 +709,11 @@
     <field name="type">url</field>
     <field name="url">/website/static/src/img/snippets_demo/s_popup.jpg</field>
 </record>
+<record id="website.s_mockup_image_default_image" model="ir.attachment">
+    <field name="public" eval="True"/>
+    <field name="name">s_mockup_image_default_image.jpg</field>
+    <field name="type">url</field>
+    <field name="url">/website/static/src/img/snippets_demo/s_mockup_image.jpg</field>
```

Comment:
Not a fan of the "website inside a website" effect.. can we use the odoo backend instead?
We discussed about the graphView, but even a nice kanban or the appSwitcher is worth considering IMO 😉